paws.common/R/iniutil.R

```diff
@@ -18,28 +18,27 @@ read_ini <- function(file_name) {
     scan(file_name, what = "", sep = "\n", quiet = T),
     perl = TRUE
   )
-  content_length <- length(content)
-
   # An empty credentials file is valid when using SSO
   # In that case, length(content) is 0.  Don't loop
   # in such a case, since 'grepl(..., content[i])'
   # will return logical(0), causing the 'if' to error out
-  if (content_length == 0) return(list())
+  if (length(content) == 0) return(list())
 
   # Get the profile name from an ini file
-  found <- grep("^\\[.*\\]", content)
   rm_els <- grep("(^;)|(^#)", content)
```

Comment:
You may want to allow for indented comments, see [this beauty of a config file](https://www.scaleway.com/en/docs/storage/object/api-cli/object-storage-aws-cli/#how-to-install-the-aws-cli) in the wild.

```suggestion
  rm_els <- grep("(^;)|(^\\s+#)", content)
```

Reply:
I will add it to the test data_ini to make sure it removes them correctly :)